elasticsearch/hooks/health-check

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
This is an elasticsearch health check, I don't think it belongs in the erlang pull request.

Reply:
Mixed up commits. I'll fix it